doc/source/quickstart.rst

```diff
@@ -0,0 +1,127 @@
+.. _my-reference-label:
+
+QuickStart
+=================
+
+Setup environment
+----------------------
+1. Python version: 3.6+
+
+2. CUDA version: 10.0+
+
+Installation and Geting Started
+--------------------------------------
+1. Since the project contains big dataset, so we use git lfs to control versoin. Please insatll git lfs first, 
+and execute
+
+.. code-block:: guess
```

Comment:
Use shell as language option even if there is no special highlight.
No need to put dollar sign.